src/index.html

```diff
@@ -3,11 +3,286 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" type="image/x-icon" href="/images/favicon.ico">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;700;800&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <h1 hidden>Bose</h1>
+    <header class="header">
```

Comment:
header is better to fit the whole screen on any device - and reveal other elements on scroll.
![header fit the screen](https://user-images.githubusercontent.com/39308717/162923689-4e8f1ccf-09e6-4f11-8757-ee767ed615e3.png)